src/main/kotlin/no/nav/syfo/personoppgave/PersonOppgaveQueries.kt

```diff
@@ -130,22 +153,113 @@ fun DatabaseInterface.createPersonOppgave(
     }
 }
 
+fun Connection.createPersonOppgave(
+    dialogmotesvar: Dialogmotesvar,
+): Pair<Int, UUID> {
+    val uuid = UUID.randomUUID().toString()
+    val now = Timestamp.from(Instant.now())
+
+    val personIdList = prepareStatement(queryCreatePersonOppgave).use {
+        it.setString(1, uuid)
+        it.setString(2, dialogmotesvar.moteuuid.toString())
+        it.setString(3, dialogmotesvar.arbeidstakerIdent.value)
+        it.setString(4, "")
+        it.setString(5, PersonOppgaveType.DIALOGMOTESVAR.name)
+        it.setTimestamp(6, now)
+        it.setTimestamp(7, Timestamp.from(dialogmotesvar.svarReceivedAt.toInstant()))
+        it.executeQuery().toList { getInt("id") }
+    }
+
+    if (personIdList.size != 1) {
+        throw SQLException("Creating person failed, no rows affected.")
+    }
+
+    return Pair(personIdList.first(), UUID.fromString(uuid))
```

Comment:
Er `Pair` en slags standard for return fra databasen? Blir egentlig returnverdien brukt noen steder?

Reply:
Du har rett, den er ikke i bruk, så vi kan heller legge det til hvis det evt trengs senere. Funksjonen returnerer nå ingenting, og så sender vi heller inn Uuid.